### DISCUSSION TITLE
Fractal Grains From Reconstruction

### DISCUSSION THREAD

**Discussion by Savillian** [mtex-toolbox/mtex#1156] [Ask Anything] [answered]
Hi everyone,

In the output of my reconstruction process, my reconstructed microstructures have begun to consist primarily of grains with fractal like orientations (see below).

![A4Y4Trial3ORToolsBetaReconstruction](https://user-images.githubusercontent.com/38051659/127025909-e3194776-8f03-474d-97d8-44be32c84b7e.png)

I'm not sure where/how this error begins, but it seems localized to prior parent grains. This leads me to think the reconstruction can recognize the distinct parent grain, but for some reason cannot plot the orientation correctly on the microstructure map.

Does anyone know of a way to address this issue?

Thanks in advance!
Alec

**Comment by kilir** [comment, marked as answer]
Hi,
your plot reminds me a bit of what is shown here:
https://github.com/mtex-toolbox/mtex/discussions/1041#discussioncomment-784858

resolved in here:
https://github.com/mtex-toolbox/mtex/discussions/1041#discussioncomment-794881

Maybe the currents version resolved your issue as well.

Cheers,
Rüdiger

**Reply by Savillian** [reply]
Hi kilir,

That did it, thank you! I was focusing on the reconstruction process, rather than the merging process, and I would never have guessed to look there.

All the best,
Alec
